### PR TITLE
Add `error` prop to proptypes for Label

### PR DESCRIPTION
**Pull request by stevesims**
**Checklist**:

* [x] Documentation
* [ ] Tests N/A
* [x] Ready to be merged

### CODE REVIEW

**Comment by penx** [comment]
Deploy preview for *govuk-react* processing.

Building with commit f54556c7fc4a1bdb07e7f87ea05df7090711e4dd

https://app.netlify.com/sites/govuk-react/deploys/5c7518f1739376000887c936

**Comment by stevesims** [comment]
Addresses specific error found in #321

File: components/label/src/index.js [hunk 1]
```diff
@@ -48,6 +48,16 @@ const Label = props => <StyledLabel {...props} />;
 Label.propTypes = {
   /** Text for the label */
   children: PropTypes.node.isRequired,
+  /**
+   * Apply error state styling to the component
+   */
+  error: PropTypes.bool,
+  // NB these propTypes don't get documented :(
```
Comment (penx) [inline components/label/src/index.js]: should this be a TODO: or issue?

Comment (stevesims) [reply components/label/src/index.js]: it's a documentation issue, relating to many/most components

so yes - we should file an issue - not having documentation for the whitespace props is a bit of an oversight :)

**Pull request merged**